lisa/sut_orchestrator/azure/features.py

```diff
@@ -211,7 +212,44 @@ def is_enabled_sriov(self) -> bool:
         return sriov_enabled
 
 
+@dataclass_json()
+@dataclass()
+class AzureNvmeOptionSettings(NvmeOptionSettings):
+    def __hash__(self) -> int:
```

Comment:
It may need to be in `NvmeOptionSettings`. It's used to compare different feature. If the disk count is not effective, it means this method should be in `NvmeOptionSettings`.

Reply:
updated